src/content/docs/network-performance-monitoring/get-started/npm-introduction.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Get started with network performance monitoring
+tags:
+  - Integrations
+  - Network performance monitoring
+  - Get started
+metaDescription: Get started with network performance monitoring in New Relic One.
+---
+
+When system performance suffers, you need to know if it’s due to your code, your infrastructure, or the underlying network. And you need to know fast, so you can focus your efforts. With network performance monitoring you can correlate and analyze application, infrastructure, digital experience, and network data all in one place, and understand how network performance and overall system performance impact each other.
+Network performance monitoring adds the context of network data to the application and infrastructure data you already collect in New Relic One. By monitoring your network data, you can:
```

Comment:
I'd break into a new paragraph here.

Reply:
I had it separated before but I hesitated, so I'll follow your advice 🙌🏻